recipe/meta.yaml

```diff
@@ -10,26 +10,19 @@ source:
 
 build:
   number: 2
-  features:
-      - vc9  # [win and py27]
-      - vc14  # [win and (py35 or py36)]
 
 requirements:
   build:
-    - python  # [win]
-    # This breals a dependency cycle:
+    - {{ compiler('c') }}
+  host:
+    # This breaks a dependency cycle:
     # curl->libssh2->cmake->curl
     - cmake-binary
```

Comment:
This can be moved to `build`.

Reply:
Maybe we can switch to `cmake` afterwards as well. Thoughts from others?

Reply:
Tried that and got something like this.

```
+ conda build /home/conda/recipe_root -m /home/conda/feedstock_root/.ci_support/linux_.yaml --quiet
BUILD START: ['libssh2-1.8.0-h5b517e9_2.tar.bz2']
libssh2 cannot depend on itself
Exited with code 1
```

Reply:
> This can be moved to build.

Done.

Reply:
Was that when it was in `host` or after moving it to `build`?

Reply:
Can't track it, so maybe, better just try it here.